Integrations/Snowflake/Snowflake.yml

```diff
@@ -0,0 +1,179 @@
+    query to retrieve new incidents
+  name: fetch_query
+  required: false
+  type: 0
+- defaultvalue: 24 hours
+  display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days)
+  name: fetch_time
+  required: false
+  type: 0
+- display: The name of the field/column that contains the datetime object or timestamp
+    for the data being fetched (case sensitive)
+  name: datetime_column
+  required: false
+  type: 0
+- display: The name of the field/column in the fetched data from which the name for
+    the demisto incident will be assigned (case sensitive)
+  name: incident_name_column
+  required: false
+  type: 0
+- defaultvalue: 'true'
+  display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+description: Your analytic data warehouse provided as Software-as-a-Service
+display: Snowflake
+name: Snowflake
+script:
+  commands:
+  - arguments:
+    - default: true
+      description: The query to execute
+      isArray: false
+      name: query
+      required: true
+      secret: false
+    - default: false
+      description: Warehouse to use for the query. Will use default if not provided.
+      isArray: false
+      name: warehouse
+      required: false
+      secret: false
+    - default: false
+      description: Database to use for the query. Will use default if not provided.
+      isArray: false
+      name: database
+      required: false
+      secret: false
+    - default: false
+      description: Schema to use for the query. Will use default if not provided.
+      isArray: false
+      name: schema
+      required: false
+      secret: false
+    - default: false
+      description: Role to use for the query. Will use default if not provided.
+      isArray: false
+      name: role
+      required: false
+      secret: false
+    - default: false
+      defaultValue: '100'
+      description: Number of rows to retrieve
+      isArray: false
+      name: rows
+      required: false
+      secret: false
+    - default: false
+      description: Optional list of columns to display in the given order
+      isArray: false
+      name: columns
+      required: false
+      secret: false
+    deprecated: false
+    description: Execute a SELECT query and retrieve the data
+    execution: false
+    name: snowflake-query
+    outputs:
+    - contextPath: Snowflake.Query
+      description: The query used to fetch results from the database
+      type: String
+    - contextPath: Snowflake.Result
+      description: Results from querying the database
+      type: Unknown
+    - contextPath: Snowflake.Database
+      description: The name of the database object
+      type: String
+    - contextPath: Snowflake.Schema
+      description: The name of the schema object
+      type: String
+  - arguments:
+    - default: true
+      description: The command to execute
```

Comment:
can we provide the optional commands ?

Reply:
Wouldn't that be limiting for the user in terms of the SQL commands they could execute - less flexible and potentially more confusing for the user?